src/javascripts/components/cards/dinos.js

```diff
@@ -0,0 +1,25 @@
+const showDinos = (array) => {
+  // document.querySelector('#add-button').innerHTML = '<button class="btn btn-success btn-lg mb-4" id="add-board-btn">Add Dinosaur</button>';
```

Comment:
Even if you plan to use this later, let's remove all commented out code. It does not belong on the production branch